rules/annotation/orf.rules

```diff
@@ -1,45 +1,41 @@
 # -*- mode: Snakemake -*-
 #
-# Viral contig annotation:
+# Contig annotation:
 # 	Rules for finding and extracting ORFs.
 #
 # See Readme.md
 
 
-from Bio.Seq import Seq
-from Bio.SeqRecord import SeqRecord
-from Bio.Alphabet import IUPAC
-
-
-rule find_genes_mga:
-    """Use MetaGene Annotator to find genes in contigs."""
+rule prodigal:
+    """Use Progial for coding genes predictions in contigs."""
     input:
         str(ASSEMBLY_FP/'{sample}_assembly'/'final-contigs.fa')
     output:
-        str(ANNOTATION_FP/'genes'/'mga'/'{sample}_contigs.genes')
+        gff = str(ANNOTATION_FP/'genes'/'prodigal'/'{sample}_genes.gff'),
+        faa = str(ANNOTATION_FP/'genes'/'prodigal'/'{sample}_genes_prot.fa'),
+        fna = str(ANNOTATION_FP/'genes'/'prodigal'/'{sample}_genes_nucl.fa'),
+        log = str(ANNOTATION_FP/'genes'/'prodigal'/'log'/'{sample}.out')
     shell:
-        """mga -m {input} > {output}"""
+        """
+        if [[ -s {input} ]]; then
+          echo "yes"
```

Comment:
Let's remove or make more verbose these debug messages (i.e. "No contigs; skipping ORF finding")

Reply:
My bad. that's only my own testing purpose. I will remove them right away.